pandasai/constants.py

```diff
@@ -8,8 +8,9 @@
 START_CODE_TAG = "<startCode>"
 END_CODE_TAG = "<endCode>"
 WHITELISTED_LIBRARIES = [
+    "pandas",
     "numpy",
-    "matplotlib",
+    "matplotlib.pyplot",
```

Comment:
Alternatively we can check for the parent module in the list. That seems neater to me than updating the list to include submodules.

Reply:
yes, agree. it would be better. I'll add the check